assets/autogpt.yaml

```diff
@@ -0,0 +1,23 @@
+---
+
+- type: application
+  name: Auto-GPT
+  organization: Auto-GPT
+  description: Auto-GPT is an experimental open-source application showcasing the
+    capabilities of the GPT-4 language model.
+  created_date: 2023-04-16
+  url: https://news.agpt.co/
+  dependencies: [GPT-4]
```

Comment:
I think the dependency here is going to be on the GPT-4 API or something like that (look at the assets under OpenAI to see the names), because these folks don't have direct access to GPT-4 directly.